recipes/django-celery-beat/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "django-celery-beat" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fdf1255eecfbeb770c6521fe3e69989dfc6373cd5a7f0fe62038d37f80f47e48
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
```

Comment:
Is pytest needed to create the package?

Reply:
I used grayskull to generate the recipes. But you're right it's possibly no needed.